commit/plugin_e2e_test.go

```diff
@@ -275,11 +275,11 @@ func setupAllNodesReadAllChains(ctx context.Context, t *testing.T, lggr logger.L
 
 		// all nodes observe the same sequence numbers lastCommittedSeqNumA for chainA and lastCommittedSeqNumB for chainB
 		n.ccipReader.On("NextSeqNum", ctx, []cciptypes.ChainSelector{chainA, chainB}).
-			Return([]cciptypes.SeqNum{lastCommittedSeqNumA, lastCommittedSeqNumB}, nil)
+			Return([]cciptypes.SeqNum{lastCommittedSeqNumA + 1, lastCommittedSeqNumB + 1}, nil)
```

Comment:
Maybe change `lastCommittedSeqNumB+1` and the like to use `nextSeqNumB` variables instead?

Reply:
imo it's fine, keeps the test more simple by having the "LastCommitedSeqNums" in the test cases